src/apps/competitions/models.py

```diff
@@ -28,14 +29,23 @@ def bundle_dataset(self):
     def get_chahub_endpoint(self):
         return "competitions/"
 
+    def clean_chahub_data(self, temp_data):
```

Comment:
Not sure if we should pull this into the main Chahub Save Mixin. Pulled it to here from profiles due to the fact logo can still be empty when we add it to the dict with a conditional.